ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/JsonRpcMethodsFactory.java

```diff
@@ -104,7 +105,12 @@ public Map<String, JsonRpcMethod> methods(
               new EeaJsonRpcMethods(
                   blockchainQueries, protocolSchedule, transactionPool, privacyParameters),
               new ExecutionEngineJsonRpcMethods(
-                  miningCoordinator, protocolContext, ethPeers, consensusEngineServer),
+                  miningCoordinator,
+                  MergeProtocolSchedule.createTimestamp(
```

Comment:
Why does another protocol schedule need to be created? Can the one from the controller be used?